src/renderer/store/modules/settings.js

```diff
@@ -242,6 +242,7 @@ const state = {
   rememberHistory: true,
   saveWatchedProgress: true,
   saveVideoHistoryWithLastViewedPlaylist: true,
+  searchCharacterLimit: 100,
```

Comment:
This is not really a setting but a constant (should not be changed by user)
But I am not sure where this should be put but probably not here

@absidue

Reply:
Constants.js? https://github.com/FreeTubeApp/FreeTube/blob/development/src/constants.js

Reply:
Done